### PR TITLE
Lily58 default keymap fix / add keymap

### PR DESCRIPTION
**Pull request by kata0510**

### CODE REVIEW

File: keyboards/lily58/keymaps/yuchi/config.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config.h"
```
Comment (drashna) [inline keyboards/lily58/keymaps/yuchi/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.

File: keyboards/lily58/keymaps/yuchi/keymap.c [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+                             _______, _______, _______, _______, _______,  _______, _______, _______ \
+  )
+};
+
+
+
+void persistent_default_layer_set(uint16_t default_layer) {
```
Comment (drashna) [inline keyboards/lily58/keymaps/yuchi/keymap.c]: Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

File: keyboards/lily58/rules.mk [hunk 3]
```diff
@@ -1,9 +1,7 @@
 SRC += matrix.c \
 	   i2c.c \
 	   split_util.c \
-	   serial.c \
-	   ssd1306.c
```
Comment (drashna) [inline keyboards/lily58/rules.mk]: Is there a reason for removing this?

File: keyboards/lily58/ssd1306.c [hunk 4]
```diff
@@ -1,330 +0,0 @@
-#ifdef SSD1306OLED
```
Comment (drashna) [inline keyboards/lily58/ssd1306.c]: Was there a reason for removing this?

**Comment by kata0510** [comment]
@drashna 
Thank you for review
Fixed the file
Deletion of ssd1306 decided not to put OLED module on Lily58.

**Comment by drashna** [comment]
Okay. Thanks for the clarification.

**Pull request merged**